api/searches.js

```diff
@@ -54,7 +54,7 @@ function search(options) {
   var config = {
     url: urls.searches(),
     query: query,
-    limit: options.limit,
+    limit: 'limit' in options ? options.limit : 10,
```

Comment:
Why is this needed? If the desired behavior is to have a default of 10, then the API should be the one enforcing that. In this case if `options.limit` is empty or null the query param is not added to the request.

Reply:
@bcvazquez this is needed to prevent `api/pager` from infinitely trying to consume pages from Data API.

This configuration value gets passed through to the pager which [defaults to `Infinity`](https://github.com/planetlabs/client/blob/f22125132ca1d6d434b69f33199d4a94e424b52b/api/pager.js#L4) and consumes all pages, so it wouldn't matter what the default for Data API is (it's 250, for reference). The way it looks right now, no options ever get added, so the pager tries to consume all results, in pages/chunks of 250 at a time from the Data API.

The change here limits the scope of the defaulting strictly to retrieving saved searches (as far as I'm aware), because while I personally think that trying to retrieve infinity anything isn't a good idea, I'm not familiar enough with use cases in the CLI and in Explorer to know if changing the value in `api/pager` would break other things.

Reply:
🤦‍♀ ... sorry Eric, I guess I didn't fully trace this nor understood the pager. Thanks for explaining it! 
And yes, api/pager gets used in other places, and this seems like the safest bet.